### PR TITLE
docs(vtkpicker): update doc and typing

### PR DESCRIPTION
**Pull request by bourdaisj**
### Changes

- [ ] Documentation and TypeScript definitions were updated to match those changes

### PR and Code Checklist

- [ ] [semantic-release](https://github.com/semantic-release/semantic-release) commit messages
- [ ] Run `npm run reformat` to have correctly formatted code

### Testing

- [ ] This change adds or fixes unit tests 
- [ ] Tested environment:
  - **vtk.js**: 
  - **OS**: 
  - **Browser**:

### CODE REVIEW

File: Sources/Rendering/Core/Picker/index.d.ts [hunk 1]
```diff
@@ -21,39 +21,34 @@ export interface vtkPicker extends vtkAbstractPicker {
...
+	getPickedPositions(): Vector3[];
 
 	/**
 	 * Get tolerance for performing pick operation.
 	 */
 	getTolerance(): number;
 
 	/**
-	 * FIXME: this needs to be check again
+	 * Invoke a pick change event.
```
Comment (finetjul) [inline Sources/Rendering/Core/Picker/index.d.ts:49]: Why remove the FIXME ? 
If there is no need to be "check again", please explain in the commit message.

File: Sources/Rendering/Core/Picker/index.d.ts [hunk 2]
```diff
@@ -127,6 +118,11 @@ export function newInstance(initialValues?: IPickerInitialValues): vtkPicker;
  * the prop itself, the data set, and the mapper. 
  * For vtkPicker the closest prop is the one whose center point (i.e., center of bounding box) projected on the view ray is closest
  * to the camera. Subclasses of vtkPicker use other methods for computing the pick point.
+ * 
+ * vtkPicker is used for quick geometric picking. If you desire more precise
```
Comment (finetjul) [inline Sources/Rendering/Core/Picker/index.d.ts:122]: `If you desire` -> use passive tense (.e.g More precise picking... must use the subclasses...)

File: Sources/Rendering/Core/Picker/index.d.ts (hunk 1 above)
Comment (daker) [reply Sources/Rendering/Core/Picker/index.d.ts:49]: the FIXME was set to check what the methods returns exactly

File: Sources/Rendering/Core/Picker/index.d.ts [hunk 3]
```diff
@@ -86,13 +81,9 @@ export interface vtkPicker extends vtkAbstractPicker {
 	setMapperPosition(x: number, y: number, z: number): boolean;
 
 	/**
-	 * Set position in mapper coordinates of pick point.
-	 * @param {Vector3} mapperPosition The mapper coordinates of pick point.
-	 */
-	setMapperPositionFrom(mapperPosition: Vector3): boolean;
```
Comment (daker) [inline Sources/Rendering/Core/Picker/index.d.ts:92]: This should not be removed, it's part of the methods
Capture d’écran du 2024-01-13 10-41-22

File: Sources/Rendering/Core/Picker/index.d.ts [hunk 4]
```diff
@@ -21,39 +21,34 @@ export interface vtkPicker extends vtkAbstractPicker {
...
 	 * Get position in mapper (i.e., non-transformed) coordinates of pick point.
 	 */
-	getMapperPosition(): number[];
+	getMapperPosition(): Vector3;
 
 	/**
-	 * 
+	 * Get a list of the points the actors returned by getActors were intersected at.
 	 */
-	getMapperPositionByReference(): number[];
```
Comment (daker) [inline Sources/Rendering/Core/Picker/index.d.ts:41]: This methods should not be removed
Capture d’écran du 2024-01-13 10-44-04

**Comment by bourdaisj** [comment]
Apologies guys @daker @finetjul , shouldn't have clicked merge.
I open another PR to fix this and check the typing of event related code: https://github.com/Kitware/vtk-js/pull/2989

**Pull request merged**